### PR TITLE
Update 102-well-architected.md

### PR DESCRIPTION
**Pull request by yashdeored**
Hi I am trying to update the AWS Well Architected Framework roadmap as it consists of old information

### CODE REVIEW

**Comment by yashdeored** [comment]
Updated the AWS Well Architected Framework roadmap as it consists of old information

**Review by dansholds — CHANGES_REQUESTED** [review]
Hey! While we appreciate the effort, the only thing that needs to be added is a mention of the sustainability pillar (the 6th pillar).

We try to keep content concise and rather than list everything ourselves its best to link out to other resources, in this case - https://aws.amazon.com/architecture/well-architected/

Please check our other topics and add this link in the same way as the others ☺️

**Comment by yashdeored** [comment]
I have updated the information can you please check it now ?

File: src/data/roadmaps/aws/content/100-introduction/103-aws/102-well-architected.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # Well Architected Framework
 
-AWS Well-Architected Framework is a set of strategic guidelines provided by Amazon Web Services. It is designed to provide high-performing and resilient systems while maintaining cost efficiency. The framework divides the architectural best practices across five pillars which include operational excellence, security, reliability, performance efficiency, and cost optimization. With this framework, you can assess and improve your cloud-based architectures and applications by leveraging AWS technologies.
\ No newline at end of file
+AWS Well-Architected Framework is a set of strategic guidelines provided by Amazon Web Services. It is designed to provide high-performing and resilient systems while maintaining cost efficiency. The framework divides the architectural best practices across six pillars which include operational excellence, security, reliability, performance efficiency, cost optimization and the newly introduced sustainability pillar. With this framework, you can assess and improve your cloud-based architectures and applications by leveraging AWS technologies. For more info :
```
Comment (dansholds) [inline src/data/roadmaps/aws/content/100-introduction/103-aws/102-well-architected.md:3]: Just keep it simple with:

"and sustainability."

File: src/data/roadmaps/aws/content/100-introduction/103-aws/102-well-architected.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
 # Well Architected Framework
 
-AWS Well-Architected Framework is a set of strategic guidelines provided by Amazon Web Services. It is designed to provide high-performing and resilient systems while maintaining cost efficiency. The framework divides the architectural best practices across five pillars which include operational excellence, security, reliability, performance efficiency, and cost optimization. With this framework, you can assess and improve your cloud-based architectures and applications by leveraging AWS technologies.
\ No newline at end of file
+AWS Well-Architected Framework is a set of strategic guidelines provided by Amazon Web Services. It is designed to provide high-performing and resilient systems while maintaining cost efficiency. The framework divides the architectural best practices across six pillars which include operational excellence, security, reliability, performance efficiency, cost optimization and the newly introduced sustainability pillar. With this framework, you can assess and improve your cloud-based architectures and applications by leveraging AWS technologies. For more info : 
+
+https://aws.amazon.com/architecture/well-architected/
```
Comment (dansholds) [inline src/data/roadmaps/aws/content/100-introduction/103-aws/102-well-architected.md:5]: This is a markdown file, so please add the link in a markdown format such as:

```- [AWS Well-Architected](https://aws.amazon.com/architecture/well-architected)```

File: src/data/roadmaps/aws/content/100-introduction/103-aws/102-well-architected.md [hunk 3]
```diff
@@ -1,3 +1,5 @@
 # Well Architected Framework
 
-AWS Well-Architected Framework is a set of strategic guidelines provided by Amazon Web Services. It is designed to provide high-performing and resilient systems while maintaining cost efficiency. The framework divides the architectural best practices across five pillars which include operational excellence, security, reliability, performance efficiency, and cost optimization. With this framework, you can assess and improve your cloud-based architectures and applications by leveraging AWS technologies.
\ No newline at end of file
+AWS Well-Architected Framework is a set of strategic guidelines provided by Amazon Web Services. It is designed to provide high-performing and resilient systems while maintaining cost efficiency. The framework divides the architectural best practices across six pillars which include operational excellence, security, reliability, performance efficiency, cost optimization and the newly introduced sustainability pillar. With this framework, you can assess and improve your cloud-based architectures and applications by leveraging AWS technologies. For more info : 
+
```
Comment (dansholds) [inline src/data/roadmaps/aws/content/100-introduction/103-aws/102-well-architected.md:4]: Please make sure you add

`Visit the following resources to learn more:`

Before the link(s)

**Review by dansholds — CHANGES_REQUESTED** [review]

**Comment by dansholds** [comment]
> I have updated the information can you please check it now ?
> 

I've left more comments, please look at the other markdown files in the repo to understand how these are structured before resubmitting the PR.

We have a style for how content links are added so please follow that.

**Review by dansholds — CHANGES_REQUESTED** [review]
Awesome, now just set the PR to merge into the `develop` branch rather than master as per the contribution guide and we're good :)

**Comment by yashdeored** [comment]
ok 👍

**Pull request closed without merging**